ligo/README.md

```diff
@@ -49,3 +49,24 @@ You can use the `Makefile` to do so as well:
 make test
 ```
 For more detail about LIGO tests, check out its [README.md](./haskell/test/)
+
+## Example contracts
+
+### RegistryDAO
+
+The storage, including configuration for the RegistryDAO
+[specification](https://github.com/tqtezos/baseDAO/blob/master/docs/registry.md)
+is included in `ligo/src/registryDAO.mligo`. You can convert this into a
+Michelson expression during the BaseDAO origination using the `ligo
+compile-storage` command as follows.
+
+```
+ligo compile-storage ligo/src/registryDAO.mligo base_DAO_contract 'default_registry_DAO_full_storage(("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), ("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), 0n, 0n, 0n, 0n, 0n)'
```

Comment:
```suggestion
is included in `src/registryDAO.mligo`. You can convert this into a
Michelson expression during the BaseDAO origination using the `ligo
compile-storage` command as follows.

```
ligo compile-storage src/registryDAO.mligo base_DAO_contract 'default_registry_DAO_full_storage(("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), ("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), 0n, 0n, 0n, 0n, 0n)'
```
I think we should remove `ligo` from the path(s) because this file is in `ligo/` already and the other commands (aka `make`) already assume we are in this directory.